### DISCUSSION TITLE
cocugu kayıp olanlar için bir sayfa

### DISCUSSION THREAD

**Discussion by sercangoger** [acikkaynak/deprem-yardim-projesi#48] [Ideas]
bi arkadaşımız cocuk kayıplarının bilgilendirmesi ve bunların teyitlerinin yapılması için bir geliştirme yapılabilmesinden söz etmiş buraya açmak istedim

**Comment by Arikitokurin** [comment]
Çok teşekkürler. Furkan Bey'in updatetoplantısında bununla ilgili bir soru gelmisti ve legalden döndüğünü anımsıyorum ancak yakın bir arkadaşımızın daçocuğu kayıp ve bulunamıyor. Twitter üzerinden de benzer haberler görüyorum. Ve resmi bir mecrayla bağlantılı şekildeyürütülmesi şart elbette umarım hızla çözüm bulunur. Tekrar çok teşekkürler.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz Legal ekibin kontrolü sonrasında reddedildi.
